partitura/io/importmusicxml.py

```diff
@@ -733,6 +733,8 @@ def _handle_direction(e, position, part, ongoing):
 
         if get_value_from_attribute(sd, "fine", str) == "yes":
             part.add(score.Fine(), position)
+        if get_value_from_attribute(sd, "fine", str) == "1":
```

Comment:
Maybe we can combine the conditions for fine (i.e., lines 734 and 736) into a single condition? Something like

```python
if get_value_from_attribute(sd, "fine", str) in ("yes", "1"):
    part.add(score.Fine(), position)
```

And maybe, if "1" is a valid value for the attributes, maybe it could come as well for "dacapo" or similar repetition symbols? (I don't know if this is the case, so we probably should not try to anticipate something that never happens). But maybe we can add a comment there for future reference in case we ever find a score where `get_value_from_attribute(sd, "dacapo", str)` has the value "1".